README.md

```diff
@@ -745,3 +745,7 @@ In another test we check for an error reply to a [bad pin :octocat:](https://git
 In tests such as [`server.test.ts` :octocat:](https://github.com/Satyam/firmata-http-bridge/blob/main/src/server.test.ts) or [`simple.test.ts` :octocat:](https://github.com/Satyam/firmata-http-bridge/blob/main/src/simple/simple.test.ts) we actually start the server and use a slightly modified version of `postCommand` to send actual HTTP request to that server, thus fully testing the app from end to end.
 
 Not everything has tests, though it can and should.  I meant to do all the tests but then, it is interesting to show how the coverage report show the code that has no tests.  Thus, I left it for homework. It is also interesting to make a test fail, by messing up with an expectation and see how a bad test is reported.
+
+### Final remarks
+
+What does *Firmata* stand for? Does it maybe come from the italian *firmare* which means *to sign*?
```

Comment:
According to Google Translate, you would be right.  Also, there is a shortage of good names.  People are getting imaginative.